packages/boxel/tests/integration/components/boxel/dropdown-test.ts

```diff
@@ -29,45 +35,77 @@ module('Integration | Component | Dropdown', function (hooks) {
     `);
   });
 
-  test('it renders dropdown trigger, but not dropdown content at initialization', async function (assert) {
-    assert.dom(TRIGGER).isVisible().containsText('Trigger');
-    assert.dom(DROPDOWN_CONTENT).doesNotExist();
+  hooks.afterEach(function () {
+    // reset Ember.onerror
+    Ember.onerror = onerror;
```

Comment:
[`setupOnerror`](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#setuponerror) and `resetOnerror` could slightly simplify this 🤓

Reply:
Yep, done, also moved the rendering error test to its own module so that the `resetOnError` call in afterEach doesn't end up being a red herring for other tests.